### DISCUSSION TITLE
Consider: add multiplatform build instructions to `./readme.md`

### DISCUSSION THREAD

**Discussion by NiloCK** [charmbracelet/bubbletea#485] [General]
Bubbletea is a nice toolkit for building tuis. One major point in Go's favour in this domain is that its built executables contain the language run-time: ie, can be run on any machine without installing language dependencies.

**BUT**, Bubbletea's documentation doesn't provide much in the way of smooth onramps to this feature. Devs building small side projects which might be of interest to other persons don't have time/energy/expertise to reproduce, for example, the contents of [your own release script](https://github.com/charmbracelet/meta/blob/main/.github/workflows/goreleaser.yml). Instead, these devs commonly provide the installation instruction `go install github/somedev/someproject`, which is intelligible only to go devs and not the broadest possible community of end users for these tui applications.

Suggestion: include in the readme a brief section with minimal instructions for either:
- a script which outputs executables for each major OS
- a github action which produces similar on version tags

Overall goal is to make it _very easy_ for hobby devs to replace their `go install ...` installation instruction line with pointers like `somedev/someproject/releases/latest`, and a brief description of what users of each environment should look for.

edit: this section would fit nicely between the existing headings [Debugging](https://github.com/charmbracelet/bubbletea#debugging) and [Libraries we use with Bubble Tea](https://github.com/charmbracelet/bubbletea#libraries-we-use-with-bubble-tea). It could be called `Building & Releasing` or some such.

**Comment by bashbunni** [comment]
Hi @NiloCK do you have an example of what this might look like? I'm not sure I understand the suggestion
